packages/core/src/cache/cache.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLSchema } from 'graphql';
+
+interface FlatbreadCache {
+  schema: GraphQLSchema | null;
+}
+
+/**
+ * A general cache for computationally heavy operations in Flatbread.
+ */
+export const cache: FlatbreadCache = {
```

Comment:
it might be worth using LRU cache or a weak map here

Reply:
Yeah I agree - I was thinking of doing something modular like this down the road:
```ts
/**
 * A general cache for computationally heavy operations in Flatbread.
 */
export const cache: FlatbreadCache = {
  schema: null,
  resolvedFields: new LRU(), // this could be hashed to the collection-node-field 
};
```
since we wouldn't want to dump the schema from an LRU cache overflowing as it's a heavier process.

With your below suggestion on keying the schema for multiple Flatbread instances, that could look something like:
```ts
/**
 * A general cache for computationally heavy operations in Flatbread.
 */
export const cache: FlatbreadCache = {
  schema: new LRU(),
  resolvedFields: new LRU(),
};
```

wdyt?

Reply:
Just added the latter